components/brave_wallet/browser/solana_transaction.cc

```diff
@@ -18,6 +18,118 @@
 
 namespace brave_wallet {
 
+namespace {
+
+constexpr char kSendOptions[] = "send_options";
+constexpr char kMaxRetries[] = "maxRetries";
+constexpr char kPreflightCommitment[] = "preflightCommitment";
+constexpr char kSkipPreflight[] = "skipPreflight";
```

Comment:
Can we add a comment saying the reason these three keys are camel style is for easy handling JSON provider request from dApps?

Reply:
https://github.com/brave/brave-core/pull/13764/commits/01421e648f43c4347572392089a8063ed95d3a1f#diff-813b4a863ad7d6dd38df6f140065b560949a28b3b73cbb16fbd00c6aa5d8d04bR25-R26